web/tests/upload.test.js

```diff
@@ -84,12 +88,21 @@ test("get_item", () => {
         $(`#edit_form_${CSS.escape(75)} .upload_banner.file_id_60`),
     );
 
+    $(`#edit_form_${CSS.escape(2)} .upload_banner`).set_find_results(
+        ".upload_banner_cancel_button",
+        $(".upload_banner_cancel_button"),
+    );
+    assert.equal(
+        upload.get_item("upload_banner_cancel_button", {mode: "edit", row: 2}, "id_34"),
+        $(`#edit_form_${CSS.escape(2)} .upload_banner.file_id_34 .upload_banner_cancel_button`),
+    );
+
     $(`#edit_form_${CSS.escape(2)} .upload_banner`).set_find_results(
         ".main-view-banner-close-button",
         $(".main-view-banner-close-button"),
     );
     assert.equal(
-        upload.get_item("upload_banner_close_button", {mode: "edit", row: 2}, "id_34"),
+        upload.get_item("upload_banner_hide_button", {mode: "edit", row: 2}, "id_34"),
         $(`#edit_form_${CSS.escape(2)} .upload_banner.file_id_34 .main-view-banner-close-button`),
     );
```

Comment:
@yogesh-sirsat  , I have replaced the name 'upload_banner_close_button' with 'upload_banner_hide_button' as it seems closer to the actual functionality of the icon, and these are the tests for it. Furthermore, I have updated the commit description accordingly. Could you please review the PR? Thanks :)

Reply:
It's confusing when you rename things, that's why we mostly do rename things work in separate commits, I'll suggest the same here too.

Reply:
@yogesh-sirsat created a separate commit for the rename, could you please review the PR, thanks :)